upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.8.yaml.template

```diff
@@ -0,0 +1,285 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cilium-config
+  namespace: kube-system
+data:
+  # This etcd-config contains the etcd endpoints of your cluster. If you use
```

Comment:
This seems to be using http etcd.  There is an update for Calico to use TLS that is going in.  You can either change the PR to output a message that you are using unencrypted etcd communication, and the etcd port is open, or figure out TLS.